suricata/update/main.py

```diff
@@ -955,6 +955,10 @@ def load_sources(suricata_version):
         if not os.path.exists(index_filename):
             logger.warning("No index exists, will use bundled index.")
             logger.warning("Please run suricata-update update-sources.")
+        if os.path.exists(index_filename) and time.time() - \
+                os.stat(index_filename).st_mtime > (60 * 60 * 24 * 14):
+            logger.warning(
+                "Index is old, please run suricata-update update-sources.")
```

Comment:
I think it should be more like:
"Source index is greater than 2 weeks old. Please update with suricata-update update-sources."